test/libsolidity/SolidityNameAndTypeResolution.cpp

```diff
@@ -5078,6 +5078,104 @@ BOOST_AUTO_TEST_CASE(invalid_address_length)
 	CHECK_WARNING(text, "checksum");
 }
 
+BOOST_AUTO_TEST_CASE(interface)
+{
+	char const* text = R"(
+		interface I {
+		}
+	)";
+	success(text);
+}
+
+BOOST_AUTO_TEST_CASE(interface_constructor)
+{
+	char const* text = R"(
+		interface I {
+			function I();
+		}
+	)";
+	success(text);
```

Comment:
The reason for having in the interface would be to mandate what constructor needs to be present. I guess we need to fix the code in order to support that.